dot/types/equivocation_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package types
+
+import (
+	"testing"
+
+	"github.com/ChainSafe/gossamer/pkg/scale"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEquivocationProof(t *testing.T) {
+	exp := []byte{
+		0, 91, 50, 25, 214, 94, 119, 36, 71, 216, 33, 152,
+		85, 184, 34, 120, 61, 161, 164, 223, 76, 53, 40,
+		246, 76, 38, 235, 204, 43, 31, 179, 28, 1, 0, 0,
+		0, 0, 0, 0, 0}
```

Comment:
I have lost the rust code that I was using to produce these bytes. Will either wait to find that code or re-write that rust code. 
After that I will be able to do tests with some extra value in BabeEquivocationProof struct.
Currently this test fails

Reply:
I like it, but do we really need this? Is this not just testing the ability to encode/decode this data? Since its not an assertion on the actual results of the runtime call, do we need it?

Reply:
replace these bytes.